flang/test/lit.cfg.py

```diff
@@ -87,6 +87,9 @@
 # the build directory holding that tool.
 tools = [
   ToolSubst('%f18', command=FindTool('f18'),
+    extra_args=["-intrinsic-module-directory "+config.flang_intrinsic_modules_dir],
+    unresolved='fatal'),
+  ToolSubst('%bbc', command=FindTool('bbc'),
```

Comment:
Is this going to break all the tests that use `bbc` and do not specify this option?

Reply:
It doesn't seem to break them. I checked the tests and they don't break, whether they use this option or not.